src/main/scala/workflow/graph/Cacher.scala

```diff
@@ -0,0 +1,24 @@
+package workflow.graph
+
+import org.apache.spark.rdd.RDD
+import pipelines.Logging
+
+import scala.reflect.ClassTag
+
+/**
+ * Caches an RDD at a given point within a Pipeline. Follows Spark's lazy evaluation conventions.
+ *
+ * @param name An optional name to set on the cached output. Useful for debugging.
+ * @tparam T Type of the input to cache.
+ */
+case class Cacher[T: ClassTag](name: Option[String] = None) extends Transformer[T,T] with Logging {
+  override protected def batchTransform(in: RDD[T]): RDD[T] = {
```

Comment:
So that we don't forget my comments on #276 - we really need to get the names right for Transformer and Estimator here.